server/src/main/java/org/spine3/server/entity/Repository.java

```diff
@@ -294,4 +307,44 @@ public int getIndex() {
             return result;
         }
     }
+
+    /**
+     * An iterator of all entities from the storage.
+     */
+    private static class EntityIterator<I, E extends Entity<I, ?>>
```

Comment:
This implementation is suitable only for `in-memory` storages. Should we say that in the documentation and recommend to override it via redefining an `#iterator`?